src/learning_item_data_retriever.py

```diff
@@ -0,0 +1,75 @@
+import json
+import pandas as pd
+from learning_item import LearningItem
+
+FOLDER_PATH = "E:\\Mi unidad\\_PYTHON\\PycharmProjects\\memoryBooster\\data\\"
+SOURCE_FILE = "data_esp.txt"
+RESULT_FILE = "data_esp2.json"
+
+def learning_items_from_file(file_path):
+    with open(file_path, 'r', encoding='utf-8') as file:
+        lines = file.readlines()
+
+    learning_items = [ ]
+    number_of_units = int(lines[0])
+    num_lines = len(lines)
+    i = 2
+    while (i < num_lines):
```

Comment:
can you extract parsing of the learning item to a separate method ? it will make it more readable.